### ISSUE TITLE
Provide optional posted notification on success

### ISSUE THREAD

**Issue by keith** [br1sk/brisk#8]
Currently if you start posting a radar with Brisk, and go to a different app, the window is closed in the background, and it's unclear whether or not the radar was posted successfully. We might want to provide a user notification fired when the posting is complete.

**Comment by ZevEisenberg** [comment]
Clicking on the notification should maybe take you to Open Radar, like it does in QuickRadar. Signing in to Open Radar manually to find the new radar is annoying.

**Comment by keith** [comment]
Seems reasonable, FWIW you can also find your new radar on the front page, since they're stored by number

**Comment by ZevEisenberg** [comment]
True, although it sure would be nice if those 8 radars at the top with the wrong number would go away. (Not a Brisk issue; just whining.)

**Comment by keith** [comment]
Haha yea definitely. Only weird part here will be when we allow people to post to services optionally each time, we'll have to handled different URLs. Should be fine.
